### ISSUE TITLE
Proper way to load non-image data?

### ISSUE THREAD

**Issue by clofresh** [leafo/aroma#4]
For example, how would I load a Tiled .tmx file? Normally you would use love.filesystem to read the file, but it doesn't look like aroma implements that api.

**Comment by leafo** [comment]
Currently there isn't a file loading api exposed, but I have an internal one used to load game resources (code and images). 

I'll keep this open, going to be making it public soon.

**Comment by bjorn** [comment]
Of course, when it is about loading a Tiled .tmx file specifically, you would instead export your map in the Lua format! :-)
